pkg/didcomm/common/model/ack.go

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package model
```

Comment:
I dislike having a namespace named "model"; however, an `ack` is neither a protocol nor a decorator, so I'm not sure where else to put it. I opened hyperledger/aries-rfc#234 to see if I get an answer.